From 6fad994185ef9c2b22824f040bd077aa6170eec1 Mon Sep 17 00:00:00 2001
From: Ran Wang <ran.wang_1@nxp.com>
Date: Thu, 11 Apr 2019 11:29:09 +0800
Subject: [PATCH 149/235] arm64: dts: ls1028a: Add USB dt nodes

This patch adds USB dt nodes for LS1028A.

Signed-off-by: Ran Wang <ran.wang_1@nxp.com>
[Xulin: Original patch taken from NXP LSDK-19.06.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 .../arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 20 +++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index a8cf92af05fb..ecaab6a245d2 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -139,6 +139,26 @@
 			clocks = <&sysclk>;
 		};
 
+		usb0:usb@3100000 {
+			compatible= "snps,dwc3";
+			reg= <0x0 0x3100000 0x0 0x10000>;
+			interrupts= <0 80 0x4>;
+			dr_mode= "host";
+			snps,dis_rxdet_inp3_quirk;
+			snps,quirk-frame-length-adjustment = <0x20>;
+			snps,incr-burst-type-adjustment = <1>, <4>, <8>, <16>;
+		};
+
+		usb1:usb@3110000 {
+			compatible= "snps,dwc3";
+			reg= <0x0 0x3110000 0x0 0x10000>;
+			interrupts= <0 81 0x4>;
+			dr_mode= "host";
+			snps,dis_rxdet_inp3_quirk;
+			snps,quirk-frame-length-adjustment = <0x20>;
+			snps,incr-burst-type-adjustment = <1>, <4>, <8>, <16>;
+		};
+
 		i2c0: i2c@2000000 {
 			compatible = "fsl,vf610-i2c";
 			#address-cells = <1>;
-- 
2.17.1

